ROADMAP.md

```diff
@@ -0,0 +1,10 @@
+# Chaos Mesh Roadmap
+
+This document defines the roadmap for Chaos Mesh development.
+
+## v0.1.0
+- [ ] Support bpf injection for time skew chaos. Simulate time jump forward or backword.
+- [ ] Add container kill/failure chaos. Support kill the special container in multi containers pod.
+- [ ] Add cpu chaos. Simulate cpu is busy.
+- [ ] Add memory chaos. Simulate can not alloc memory.
```

Comment:
cpu and memory chaos seems like `node` level chaos. Maybe we could support and extend more abilities and methods of selector for existed chaos first.

Reply:
How about "Make scheduler optional"

Reply:
How about "Make scheduler optional"

Reply:
@zhouqiang-cl In my view, `Make scheduler optional` is necessary for v1.0 which would make `chaos api` both user-friendly and developer-friendly.

Reply:
Yes. already add it below

Reply:
what do you mean about `selector`, did it means select of pod? seems we have already support it